org.phoenicis.playonlinux.yml

```diff
@@ -0,0 +1,67 @@
+---
+app-id: org.phoenicis.playonlinux
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
```

Comment:
It should be quoted, otherwise it's not a string. (Also everywhere below where applicable.)

Reply:
Are you sure? I followed the example in [YAMLing the flathub](https://blogs.gnome.org/alexl/2018/05/04/yamling-the-flathub/). Also: if it is really wrong, why is the build working?

Reply:
It's coincidence depending on implementation detail. Versions are internally a string, not floats.

Reply:
Ok.